app/src/main/java/com/sunasterisk/travelapp/data/models/Reservation.kt

```diff
@@ -1,21 +1,21 @@
 package com.sunasterisk.travelapp.data.models
 
-class Reservation(
+data class Reservation(
     val id: Int = -1,
     val restaurant: Restaurant,
     val email: String,
     val phone: String,
     val name: String,
     var arrivalTime: String,
     var numAdult: Int,
-    var numChildrent: Int
+    var numChildren: Int
 ) {
     companion object {
         const val TABLE_NAME = "reservation"
         const val ID = "Reservation_Id"
```

Comment:
```suggestion
        const val ID = "reservation_id"
```
tương tự các thuộc tính còn lại